guides/release/upgrading/editions.md

```diff
@@ -126,103 +128,148 @@ attributes can be used _without_ setting a value at all, just like HTML
 attributes. The component you are invoking decides where to put these attributes
 by using the special `...attributes` syntax. This will be discussed later in
 the section on components.
+
 For classic components, only attributes that were explicitly listed by the component
 you are invoking would be placed on the component's wrapper element.
 
-#### Getting used to Angle Brackets
+- _Arguments_ and _attributes_ are distinguished from each other when using a
+  component. With curly brace style components, every value you pass to the
+  component is an _argument_ - a JS value that is passed to the component class
+  so it can be used in the component's template:
 
-Here is what is different about using Angle Bracket syntax.
+  ```handlebars
+  {{!-- In this example, `value` is an argument --}}
+  {{input value=this.text}}
```

Comment:
Not sure if `{{input` is a good example, since it's not really a component since no dash.

Reply:
Plus `@value` vs `value` comes up right away since it's built in.